### DISCUSSION TITLE
Deploying on vercel

### DISCUSSION THREAD

**Discussion by Brkic365** [quirrel-dev/quirrel#1023] [Q&A]
Hello there! I used Quirrel on my project and it worked awesome locally, but now I want to deploy my application. Can you help me get the QUIRREL_TOKEN? I tried following the link (https://quirrel.dev/dashboard) but it leads me to log in, and I don’t have an account. I am pretty new to Quirrel.

**Comment by rodrigoimenes** [comment]
Hello @Brkic365, you can follow the docs here https://docs.quirrel.dev/deploying#acquire-a-token
You only need to run it against your quirrel server :)
